Description: Fix typo and update for new FreeDesktop standard
 Encoding key is now deprecated by the FreeDesktop standard.
Author: Osamu Aoki <osamu@debian.org>

--- a/client/x11/main.c
+++ b/client/x11/main.c
@@ -301,7 +301,7 @@
     }
 
     for (i=0; i< (int) call_data->status_attr_num; ++i, ++sts_attr) {
-        LOG (1, "Unkown status attribute: %s", sts_attr->name);
+        LOG (1, "Unknown status attribute: %s", sts_attr->name);
     }
 
     return attrs;
@@ -720,7 +720,7 @@
         LOG (1, "XIM_SYNC_REPLY");
         return 0;
     default:
-        LOG (1, "Unkown (%d)", call_data->major_code);
+        LOG (1, "Unknown (%d)", call_data->major_code);
         return 0;
     }
 }
--- a/setup/ibus-setup.desktop.in
+++ b/setup/ibus-setup.desktop.in
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=IBus Preferences
 _Comment=Set IBus Preferences
 Exec=ibus-setup
--- a/bus/ibus.desktop.in
+++ b/bus/ibus.desktop.in
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 _Name=IBus
 _GenericName=Input Method Framework
 _Comment=Start IBus Input Method Framework
